userspace/libsinsp/filterchecks.cpp

```diff
@@ -3537,6 +3661,12 @@ uint8_t* sinsp_filter_check_thread::extract(sinsp_evt *evt, OUT uint32_t* len, b
 	case TYPE_IS_EXE_FROM_MEMFD:
 		m_tbool = tinfo->m_exe_from_memfd;
 		RETURN_EXTRACT_VAR(m_tbool);
+	case TYPE_IS_SID_LEADER:
+		m_tbool = tinfo->m_sid == tinfo->m_vpid ? true : false;
```

Comment:
Super nit, feel free to skip this :D 
```suggestion
		m_tbool = tinfo->m_sid == tinfo->m_vpid;
```

Same for `TYPE_IS_VPGID_LEADER` below!